recipes/crossguid/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.apple import is_apple_os
+import os
+
+required_conan_version = ">=1.53.0"
+
+class CrossGUIDConan(ConanFile):
+    name = "crossguid"
+    description = "Lightweight cross platform C++ GUID/UUID library"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/graeme-hill/crossguid"
+    topics = ("guid", "uuid")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.settings.os == "Linux":
+            self.requires("libuuid/1.0.3", transitive_headers=True, transitive_libs=True)
```

Comment:
Shouldn't it be replaced by `util-linux-libuuid` since all other recipes depend on it instead of `libuuid` recipe (and both recipes can't live together in the same graph since there is a provides attribute in `util-linux-libuuid`)?

Reply:
I've opened https://github.com/conan-io/conan-center-index/pull/20902

Reply:
Moreover traits of libuuid were wrong. These undefined references come from a mistake in this CMake patch where you don't properly link libuuid actually (there is no such thing as libuuid_LIBRARY in CMakeDeps)